### PR TITLE
Add matter auto suggest input

### PR DESCRIPTION
**Pull request by erlikh**

### CODE REVIEW

**Comment by ntgn81** [comment]
Nice work! I tried to customize this thing to fit in with the other inputs, seems like this plugin does not give a lot of hooks for that. It's wrapping a lot of things together. 

It doesn't work quite well with the theming that we have. Do you want to try and see if it's easy to do?

In the meantime, I checked out `react-autosuggest`. It's minimal mark up might be a better fit:D

**Comment by ntgn81** [comment]
Something like this

```js
function getSuggestions (value, cb) {
  cb(null, ['a', 'b', 'c']);
}
```

```jsx
<div className='form-group form-group-sm'>
  <label>Autosuggest</label>
  <Autosuggest
    suggestions={getSuggestions}
    inputAttributes={{className: 'form-control'}} />
</div>
```

**Comment by erlikh** [comment]
Thanks!
I'll definitely try the `react-autosuggest`. Also I'll try to style the current thing.

File: src/components/MatterSuggestContainer.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+  render () {
+    let clientMatters;
+    try {
+      clientMatters = this.props.matters[this.props.clientId].map(matter => matter.matterNumber);
+    } catch (_) {
+      return null;
+    }
+    return <div className='form-group form-group-sm'>
+      <label htmlFor='matters-autosuggest'>Two-Way Related Matters</label>
```
Comment (ntgn81) [inline src/components/MatterSuggestContainer.js]: This now makes it specific to two-way related matters box. I think we want something more generic

Comment (ntgn81) [inline src/components/MatterSuggestContainer.js]: But this is fine for now, we'll optimize later

Comment (erlikh) [inline src/components/MatterSuggestContainer.js]: sure

**Pull request merged**